.circleci/config.yml

```diff
@@ -17,6 +17,19 @@ executors:
     docker:
       - image: google/cloud-sdk:alpine
 
+anchors:
+  # Run jobs on PR branches only
```

Comment:
This comment isn't always true since you can place an anchor in multiple locations. I'd remove both of these, since the implementation speaks for itself.